cmd/snap/color.go

```diff
@@ -37,11 +37,11 @@ type unicodeMixin struct {
 func (ux unicodeMixin) addUnicodeChars(esc *escapes) {
 	if canUnicode(ux.Unicode) {
 		esc.dash = "–" // that's an en dash (so yaml is happy)
-		esc.uparrow = "↑"
+		esc.rightUpArrow = "⬏"
 		esc.tick = "✓"
 	} else {
 		esc.dash = "--" // two dashes keeps yaml happy also
-		esc.uparrow = "^"
+		esc.rightUpArrow = "^"
```

Comment:
Seeing as we're editing this line, anyhow, would the following suggestion help to improve the understanding of its meaning at all?

```suggestion
		esc.rightUpArrow = "-^"
```